src/routes/(app)/+layout.svelte

```diff
@@ -12,7 +12,7 @@
 
   <meta charset="utf-8" />
   <title>Login Application</title>
-  <link rel="icon" href="/favicon.ico" />
+  <link rel="icon" href="/favicon.png" />
```

Comment:
honestly this still throws but i have no idea how the favicon was being picked up to begin with.